### ISSUE TITLE
Review visual indication of links for accessibility

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/dimes#200]
## Is your feature request related to a problem? Please describe.
[WCAG 2.1 SC 1.4.1 ](https://www.w3.org/TR/WCAG21/#use-of-color)specifies that color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element. 

## Describe the solution you'd like
Review link text in regards to [WCAG 2.1 AA recommendations](https://www.w3.org/WAI/WCAG21/Techniques/general/G183). Where color alone is used to identify them, the contrast ratio should be 3:1. Consider underlining links in contexts such as the Request Copies modal and modal confirmation messages.
